Mathlib/CategoryTheory/Preadditive/EilenbergMoore.lean

```diff
@@ -0,0 +1,198 @@
+/-
+Copyright (c) 2022 Julian Kuelshammer. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Julian Kuelshammer
+
+! This file was ported from Lean 3 source module category_theory.preadditive.eilenberg_moore
+! leanprover-community/mathlib commit 829895f162a1f29d0133f4b3538f4cd1fb5bffd3
+! Please do not edit these lines, except to modify the commit id
+! if you have ported upstream changes.
+-/
+import Mathlib.CategoryTheory.Preadditive.Basic
+import Mathlib.CategoryTheory.Monad.Algebra
+import Mathlib.CategoryTheory.Preadditive.AdditiveFunctor
+
+/-!
+# Preadditive structure on algebras over a monad
+
+If `C` is a preadditive category and `T` is an additive monad on `C` then `Algebra T` is also
+preadditive. Dually, if `U` is an additive comonad on `C` then `Coalgebra U` is preadditive as well.
+
+-/
+
+
+universe v₁ u₁
+
+-- morphism levels before object levels. See note [category_theory universes].
```

Comment:
```suggestion
-- morphism levels before object levels. See note [category_theory universes].
```
I think it's okay to just drop these comments, but if you keep them they should be *before* the `universe` line above. For some reason `mathport` puts it later.